Makefile

```diff
@@ -31,7 +31,7 @@
 include config/version
 
 SHELL = /bin/sh
-SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/[0-9\.]//g;')
+SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/\.//g' -e 's/[0-9]\{1,\}.*//')
```

Comment:
I used basic REs here since that's what was being used. It could be written more succinctly with extended REs.

I removed `;` because that is used to separate multiple sed expressions within a single arg and there is only one sed expression (now two, but only one in each arg).